pandas/tests/indexes/common.py

```diff
@@ -298,7 +298,7 @@ def test_ensure_copied_data(self, indices):
         elif isinstance(indices, (RangeIndex, MultiIndex, CategoricalIndex)):
             # RangeIndex cannot be initialized from data
             # MultiIndex and CategoricalIndex are tested separately
-            return
+            pytest.skip()
```

Comment:
can you add a message to the skip

Reply:
I'll just remove them from this PR, as it's not really connected. It was rather a left-over from the initially suggested changes